src/aggregation/models/aggregationManager.ts

```diff
@@ -46,7 +47,12 @@ export class AggregationManager {
           const aggregationMetadataLayer = schemaParser({ schema: aggregationMetadataSchema, value: aggregationResult });
           return aggregationMetadataLayer;
         } catch (error) {
-          const errorMessage = `Could not aggregate polygon parts`;
+          if (error instanceof ValidationError) {
+            const errorMessage = 'Invalid aggregation metadata response';
+            logger.error({ msg: errorMessage, error });
+            throw new InternalServerError(errorMessage);
```

Comment:
You don't have to specific throw InternalServerError as it the the error handler default error type

Reply:
done